solr/core/src/test/org/apache/solr/rest/TestManagedResourceStorage.java

```diff
@@ -116,7 +103,7 @@ private void doStorageTests(SolrResourceLoader loader, StorageIO storageIO) thro
     assertEquals("true", storedArgs.get("ignoreCase"));
     List<String> storedList = (List<String>)storedMap.get(ManagedResource.MANAGED_JSON_LIST_FIELD);
     assertNotNull(storedList);
-    assertTrue(storedList.size() == managedList.size());
+    assertEquals(storedList.size(), managedList.size());
```

Comment:
nit: I think managedList is the expected value, swap the order of these arguments.